src/components/Editor/Editor.css

```diff
@@ -86,6 +86,13 @@ html[dir="rtl"] .editor-mount {
   mask-size: auto 12px;
   mask-position: right;
 }
+.theme-light .cm-s-mozilla .empty-line .CodeMirror-linenumber,
+.theme-dark .cm-s-mozilla .empty-line .CodeMirror-linenumber {
```

Comment:
Ooops, this second line needs to be removed since you're setting the dark theme color below it.